Tests/TuistLoaderTests/Up/UpMintTests.swift

```diff
@@ -0,0 +1,158 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistGenerator
+import TuistSupport
+import XCTest
+
+@testable import TuistLoader
+@testable import TuistSupportTesting
+
+final class UpMintTests: TuistUnitTestCase {
+    var upHomebrew: MockUp!
+
+    override func setUp() {
+        super.setUp()
+        upHomebrew = MockUp()
+    }
+
+    override func tearDown() {
+        upHomebrew = nil
+        super.tearDown()
+    }
+
+    func test_init() throws {
```

Comment:
I'd add comments to the test bodies to be consistent with the convention that we have been following in the codebase:

```swift
func test_something() {
  // Given
  // ... code to set up the scenario

  // When
  // ... the action on the test subject

  // Then
  // ... the expectations
}
```

Reply:
Sure 👍